### DISCUSSION TITLE
you haven't been granted specific permission to do anything

### DISCUSSION THREAD

**Discussion by abrozzi** [kjolley/BIGSdb#856] [General]
```
Database curator's interface - Isolates

Although you are set as a curator/submitter, you haven't been granted specific permission to do anything. Please contact the database administrator to set your appropriate permissions.
```

I created a user in USERS for the database bigsdb_test_isolates:

```
INSERT INTO USERS (user_name,surname,first_name,email,affiliation, date_entered,datestamp,status, curator) VALUES ('kjolley','Jolley','Keith','keith.jolley@zoo.ox.ac.uk','University of Oxford, UK','now','now','validated',1)
```

And the run add_user.pl.

I tried to set curator as 5 but it doesn't work. It accepts only 0 and 1.

Can you tell me this is the correct procedure to set one user?

The manual only show the details for site-wide.

Thank you!

**Comment by kjolley** [comment]
The only valid values for status here are 'admin', 'curator', 'submitter', or 'user'. You need to set this to 'admin', not 'validated'.

https://bigsdb.readthedocs.io/en/latest/dbase_setup.html#setting-up-the-admin-user

The curator field is the id of the user creating the user. It needs to exist in the users table.
